Vizzuality/trase
# Feature/map interaction

## Description

This PR:

- Adds click and hover interaction
- Fetches new pbf unit layers from the services endpoint.
- Fetches the new unit layer uploaded data from Carto DB
- Adds this unit layers data to the map tooltips. This data depends on the years selected.

- There are some regions without data and they don't show any tooltip.

Missing:
- Soy yield indicator is giving accumulated data but it differs from the value on the Sankey. We might have to change the Sankey logic.

![image](https://user-images.githubusercontent.com/9701591/82828192-d13ba080-9eb0-11ea-891f-93e66f587018.png)

## Testing instructions

- Open different map contexts (Brazil-Soy), change the unit layers (On the layers modal) or change the Sankey columns (Municipalities => States)
- The map should have a tooltip with interaction when you hover and the hovered region should be highlighted in black. 
- The selected regions on the map should be selected on the tooltip and vice versa. The selected regions on the map should be highlighted in black.
